contents/docs/self-host/configure/securing-posthog.mdx

```diff
@@ -69,14 +69,12 @@ make it is secure to threat actors. As every installation is unique, we provide
 in our documentation some general guidelines regarding how to make it secure but
 it is then up to each instance admin to make sure all the best practices are followed.
 
-1. make sure your Kubernetes cluster is provisioned and configured using industries
+1. Make sure your Kubernetes cluster is provisioned and configured using industries
```

Comment:
remove the `## Securing PostHog Deployments` that's tied to k8s. The rest of the envs are relevant for others too.